binding/springhttpclient/src/test/java/io/tracee/binding/springhttpclient/TraceeClientHttpInterceptorSpringContextIT.java

```diff
@@ -0,0 +1,87 @@
+package io.tracee.binding.springhttpclient;
+
+import io.tracee.Tracee;
+import io.tracee.TraceeConstants;
+import org.eclipse.jetty.server.Handler;
+import org.eclipse.jetty.server.Request;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.server.handler.AbstractHandler;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ErrorCollector;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.web.client.RestTemplate;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.net.InetSocketAddress;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+/**
+ * Spring configuration test.
+ */
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration("classpath:/test-context.xml")
+public class TraceeClientHttpInterceptorSpringContextIT {
+
+
+	private static final class EmptyEntity {}
+
+	private Server server;
+	private String serverEndpoint;
+
+	@Autowired @Qualifier("restTemplate")
```

Comment:
Yeah but It ensures, that it will not pull another random `RestTemplate` from the spring scope that may have another name. Just a painful learned lesson from my daily work but I agree, we could leave it in this case